src/widgets/helper/ChannelView.cpp

```diff
@@ -1640,13 +1640,15 @@ void ChannelView::handleMouseClick(QMouseEvent *event,
 }
 
 void ChannelView::addContextMenuItems(
-    const MessageLayoutElement *hoveredElement, MessageLayout *layout)
+    const MessageLayoutElement *hoveredElement, MessageLayoutPtr layout)
 {
     const auto &creator = hoveredElement->getCreator();
     auto creatorFlags = creator.getFlags();
 
-    static QMenu *menu = new QMenu;
-    menu->clear();
+    auto menu = new QMenu;
+    connect(menu, &QMenu::aboutToHide, [menu] {
+        menu->deleteLater();  //
```

Comment:
Here.

Reply:
Correct, this is just to avoid funky auto-formatting by `clang-format`

Without the trailing `//`, the `connect` statement becomes a single line
`    connect(menu, &QMenu::aboutToHide, [menu] { menu->deleteLater(); });`

Reply:
Okay, just making sure it wasn't missed by accident. 😄